.mage/git.go

```diff
@@ -42,7 +43,7 @@ func (Git) installHook(name string) (err error) {
 			err = closeErr
 		}
 	}()
-	if _, err = fmt.Fprintf(f, "ARGS=\"$@\" make git.%s\n", name); err != nil {
+	if _, err = fmt.Fprintf(f, "HOOK=\"%s\" ARGS=\"$@\" \"%s\" run github.com/magefile/mage git:runHook\n", name, mg.GoCmd()); err != nil {
```

Comment:
I think we should just keep doing this with Make.
If you really think this is better, then at least call `./mage`. I don't want to `go run` (compile) Mage on every git commit.